core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/HealthMonitor.scala

```diff
@@ -94,6 +95,18 @@ class HealthMonitor private (val slickDataSource: SlickDataSource, val googleSer
   // not interfere with user facing operations.
   import context.dispatcher
 
+    /**
+    * A monoid used for combining SubsystemStatuses.
+    * Zero is an ok status with no messages.
+    * Append uses && on the ok flag, and ++ on the messages.
+    */
+  private implicit val SubsystemStatusMonoid = new Monoid[SubsystemStatus] {
+    def combine(a: SubsystemStatus, b: SubsystemStatus): SubsystemStatus = {
+      SubsystemStatus(a.ok && b.ok, a.messages |+| b.messages)
+    }
+    def empty: SubsystemStatus = OkStatus
+  }
+
```

Comment:
I moved this up because scalac was complaining that it couldn't find the implicit later on.